src/charm.py

```diff
@@ -136,11 +135,11 @@ def _get_version(self) -> Optional[str]:
             return
         return version
 
-    def tempo_otlp_grpc_endpoint(self) -> Optional[str]:
+    def tempo_otlp_http_endpoint(self) -> Optional[str]:
         """Endpoint at which the charm tracing information will be forwarded."""
         # the charm container and the tempo workload container have apparently the same
         # IP, so we can talk to tempo at localhost.
-        return f"http://localhost:{self.tempo.otlp_grpc_port}/"
+        return f"http://localhost:{self.tempo.otlp_http_port}/v1/traces"
```

Comment:
should we be using https here btw? I don't think we have tls on Tempo yet, perhaps we should add a TODO here for when that's done?

Reply:
Added a TODO in here.